billy-core/src/main/java/com/premiumminds/billy/core/exceptions/SeriesUniqueCodeNotFilled.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017 Premium Minds.
+ *
+ * This file is part of billy core.
+ *
+ * billy core is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or (at your option) any
+ * later version.
+ *
+ * billy core is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with billy core. If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.premiumminds.billy.core.exceptions;
+
+public class SeriesUniqueCodeNotFilled extends Exception {
+
+	private static final long serialVersionUID = 1L;
+
+	public SeriesUniqueCodeNotFilled(String message) {
+		super(message);
+	}
+
+	public SeriesUniqueCodeNotFilled(Throwable cause) {
```

Comment:
Out of all the implemented constructors only one of them is actually used. From the perspective of reducing unused code perhaps we could leave only the relevant code.